service/history/ndc/workflow_resetter.go

```diff
@@ -930,11 +930,26 @@ func reapplyEvents(
 				continue
 			}
 			attr := event.GetWorkflowExecutionOptionsUpdatedEventAttributes()
+			callbacks := attr.GetAttachedCompletionCallbacks()
+			if len(callbacks) > 0 {
```

Comment:
Want to understand if we can only verify whether the requestID exists or not. Is there any case where requestID can be attached but not all of the callbacks?